changelog.yml

```diff
@@ -1,6 +1,27 @@
 name: pypi-mobans
 organisation: moremoban
 releases:
+  - changes:
+      - action: Updated
+        details:
+          - sync templates/docs/make.bat.jj2 to include changes from
+            https://github.com/sphinx-doc/sphinx/commit/0dbdae31
+          - "#100: Fixed travis.yml.jj2 to not emit yamllint errors"
```

Comment:
I see I need to split this up into multiple action groups

Reply:
I split up https://github.com/moremoban/moban/pull/293 a bit more.  Is that better?

If so, I can split this one up.

Feel free to fiddle with them, if you think that will be faster.

Reply:
Actually this one looks great! :)